dapr/clients/grpc/_response.py

```diff
@@ -62,58 +72,76 @@ class InvokeServiceResponse(DaprResponse):
     and protocol buffer data.
 
     Attributes:
-        data (:obj:`google.protobuf.any_pb2.Any`): the serialized protocol
+        headers (tuple, optional): the tuple for the headers from response.
```

Comment:
should it be MetadataTuple ?

Reply:
I would use python primitive type instead of hinting type for now. I will reiterate all comments later to update comments.

Reply:
same here. MetadataTuple is the alias of Tuple.

Reply:
I may need to review all comments later. :)

Reply:
I understand that it is an internal type but in the docs can it be (tuple(tuple(str, (str, bytes)))) ?or something similar ?